sys-auth/sssd/sssd-1.16.3.ebuild

```diff
@@ -0,0 +1,208 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
you do not need this version at all

Reply:
Why not? Last version in previous major + current major